packages/labs/test/selectTests.tsx

```diff
@@ -98,39 +98,127 @@ describe("<Select>", () => {
         assert.strictEqual(wrapper.state("query"), "");
     });
 
+    it("if query is non-empty, the input value will stay in sync with the query prop", () => {
```

Comment:
"if query [prop] is non-empty" ? can't tell what you're trying to achieve from title.